sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerTransformRegistrar.java

```diff
@@ -341,6 +346,9 @@ public PTransform<PCollection<Row>, PDone> buildExternal(
               .withDatabaseId(configuration.databaseId)
               .withInstanceId(configuration.instanceId);
 
+      if (configuration.highPriority != null && configuration.highPriority) {
```

Comment:
is it worth defaulting to false just so you can avoid a null check?

Reply:
I agree. Zero value should just be false. Thank you Pablo for doing this :-)